src/services/add-query-to-queue.ts

```diff
@@ -5,15 +5,25 @@ import {inject, injectable} from 'inversify';
 import shuffle from 'array-shuffle';
 import {TYPES} from '../types.js';
 import GetSongs from '../services/get-songs.js';
-import {SongMetadata, STATUS} from './player.js';
+import {MediaSource, SongMetadata, STATUS} from './player.js';
 import PlayerManager from '../managers/player.js';
 import {buildPlayingMessageEmbed} from '../utils/build-embed.js';
 import {getMemberVoiceChannel, getMostPopularVoiceChannel} from '../utils/channels.js';
 import {getGuildSettings} from '../utils/get-guild-settings.js';
+import {SponsorBlock} from 'sponsorblock-api';
+import Config from './config';
 
 @injectable()
 export default class AddQueryToQueue {
-  constructor(@inject(TYPES.Services.GetSongs) private readonly getSongs: GetSongs, @inject(TYPES.Managers.Player) private readonly playerManager: PlayerManager) {
+  private readonly sbClient?: SponsorBlock;
+  private sbClientTimeout = false;
+
+  constructor(@inject(TYPES.Services.GetSongs) private readonly getSongs: GetSongs,
+    @inject(TYPES.Managers.Player) private readonly playerManager: PlayerManager,
+    @inject(TYPES.Config) private readonly config: Config) {
+    this.sbClient = config.ENABLE_SPONSORBLOCK
```

Comment:
personally I try to avoid abbreviations for anything that's not incredibly common (e.g. `req`/`res`), so maybe
```suggestion
    this.sponsorBlock = config.ENABLE_SPONSORBLOCK
```
?